requirements/pip.txt

```diff
@@ -0,0 +1,2 @@
+pip==20.2.3
```

Comment:
Do we have a pip.txt file in other services?

Reply:
we've started to introduce this across services with python 3.8 update for Dockerfile in services
ref: https://github.com/edx/registrar/pull/354#discussion_r497710581